src/main/java/eu/interop/federationgateway/entity/DiagnosisKeyCleanupEntity.java

```diff
@@ -0,0 +1,73 @@
+/*-
+ * ---license-start
+ * EU-Federation-Gateway-Service / efgs-federation-gateway
+ * ---
+ * Copyright (C) 2020 - 2022 T-Systems International GmbH and all other contributors
+ * ---
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ---license-end
+ */
+
+package eu.interop.federationgateway.entity;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * This class represents the Batch - entity.
+ */
+@Schema(
```

Comment:
As this entity is not exposed to the REST API, I would suggest not to have a swagger documentation here. Same for the other new entities

Reply:
Done